src/pages/problems/index.tsx

```diff
@@ -4,7 +4,9 @@ import { createGetServerSideProps } from '../../utils/create-get-ssr-props';
 export const getServerSideProps = createGetServerSideProps({
     requireSession: true,
     action: async ({ ssg }) => {
-        await ssg.problems.getList.fetch({});
+        await ssg.problems.getList.fetchInfinite({
```

Comment:
Does it connected to SettingsPage issue?

Reply:
Oops, deleted